public/app/features/explore/LiveLogs.test.tsx

```diff
@@ -19,68 +40,41 @@ describe('LiveLogs', () => {
       />
     );
 
-    expect(wrapper.contains('log message 1')).toBeTruthy();
-    expect(wrapper.contains('log message 2')).toBeTruthy();
-    expect(wrapper.contains('log message 3')).toBeTruthy();
-  });
+    expect(screen.getByRole('cell', { name: 'log message 1' })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: 'log message 2' })).toBeInTheDocument();
+    expect(screen.getByRole('cell', { name: 'log message 3' })).toBeInTheDocument();
```

Comment:
This wasnt part of the original test, but we might want to check that 4, 5 and 6 are not visible here as well.